### ISSUE TITLE
maplibre-shield-generator 0.0.3-test-1 incompatible with latest style

### ISSUE THREAD

**Issue by 1ec5** [ZeLonewolf/openstreetmap-americana#957]
#941 made backwards-incompatible changes to the `banneredShield()` function that the style now relies on. When I updated the AARoads fork to incorporate this change in aaroads-wiki/openstreetmap-americana@441c25aa942297846863b473000c3ed2cba567ac, [embedded maps](https://wiki.aaroads.com/wiki/Prince_William_Parkway) on the AARoads Wiki began to show generic shields and errors:

<img src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/316c80b7-d4b1-4852-bb7a-1b0cd91a5462" width="290" alt="Virginia State Route 294 as a rectangle">

```
Exception while loading image ‘shield
US:VA=123’:
 TypeError: t is undefined
    _t shield_renderer.ts:191
    he shield_renderer.ts:191
    Xe shield_renderer.ts:191
    getStyleImageMissingHandler shield_renderer.ts:191
    fire Prince_William_Parkway line 2 > injectedScript:31
    fire Prince_William_Parkway line 2 > injectedScript:31
    fire Prince_William_Parkway line 2 > injectedScript:31
    _notify Prince_William_Parkway line 2 > injectedScript:35
    getImages Prince_William_Parkway line 2 > injectedScript:35
    getImages Prince_William_Parkway line 2 > injectedScript:35
    processTask Prince_William_Parkway line 2 > injectedScript:31
    receive Prince_William_Parkway line 2 > injectedScript:31
shield_renderer.ts:191:22
    getStyleImageMissingHandler shield_renderer.ts:191
    fire evented.ts:121
    fire evented.ts:136
    fire evented.ts:136
    _notify image_manager.ts:206
    getImages image_manager.ts:193
    getImages style.ts:1537
    processTask actor.ts:197
    receive actor.ts:136
```

The OSM and AARoads wikis both depend on maplibre-shield-generator 0.0.3-test-1. We’ll need a new release of the @americana/maplibre-shield-generator library. #889 will make this kind of breakage less likely by forcing us to explicitly update this repository whenever it depends on something new in the library.

**Comment by ZeLonewolf** [comment]
Just published [0.0.4](https://www.npmjs.com/package/@americana/maplibre-shield-generator/v/0.0.4) which adds the new banner features.

**Comment by 1ec5** [comment]
Thanks, it’s looking good on both AARoads Wiki and OSM Wiki.

<img src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/c3538a7c-1737-4189-bd25-ce83c4f7ab8d" width="290" alt="Virginia State Route 294">
